test/test_app_helpers.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path(File.join(File.dirname(__FILE__), "helper"))
+
+context 'Precious::Helpers' do
+  include Precious::Helpers
+  
+  test 'remove trailing and leading slashes' do
```

Comment:
This test could also be removed, I just used it to make sure I was getting the regexes right :)

Reply:
I don't mind leaving this in; up to you.